lib/web_ui/lib/src/engine/canvaskit/initialization.dart

```diff
@@ -86,7 +91,7 @@ const bool canvasKitForceCpuOnly = bool.fromEnvironment(
 /// NPM, update this URL to `https://unpkg.com/canvaskit-wasm@0.34.0/bin/`.
 const String canvasKitBaseUrl = String.fromEnvironment(
   'FLUTTER_WEB_CANVASKIT_URL',
-  defaultValue: 'https://unpkg.com/canvaskit-wasm@0.28.1/bin/',
+  defaultValue: 'https://unpkg.com/canvaskit-wasm@$canvaskitVersion/bin/',
```

Comment:
FYI @jacob314 you will probably need to update the devtools code that reads the version from here

Reply:
Can devtools read it from `canvaskit_lock.yaml`?

Reply:
sure that would be fine.

Reply:
Thanks for confirming!